tests/dummy/app/templates/components/page-toolbar.hbs

```diff
@@ -1,7 +1,7 @@
 {{#paper-toolbar class="md-whiteframe-1dp page-main-toolbar" as |toolbar|}}
   {{#toolbar.tools}}
-    {{#paper-sidenav-toggle as |toggle|}}
-      {{#paper-button onClick=(action "toggleMenu" target=toggle) iconButton=true}}
+    {{#paper-sidenav-toggle class="hide-gt-sm"}}
+      {{#paper-button onClick=null iconButton=true}}
```

Comment:
Is this idiomatic? Maybe it should still be:

```hbs
{{#paper-sidenav-toggle as |toggle|}}
  {{#paper-button onClick=toggle iconButton=true}}
    {{paper-icon "menu"}}
  {{/paper-button}}
{{/paper-sidenav-toggle}}
```

That way you can also use a link or something instead and it's explicit what the purpose of the toggle+button combo is?